amostra/tests/test_jsonschema.py

```diff
@@ -0,0 +1,40 @@
+import hypothesis_jsonschema
+from hypothesis import given, settings
+from hypothesis import strategies as st
+
+from amostra.utils import load_schema
+
+sample_dict = load_schema("sample.json")
+# Pop uuid and revision cause they are created automatically
+sample_dict['properties'].pop('uuid')
+sample_dict['properties'].pop('revision')
+sample_dict['required'] = ['name']
+sample_dict['properties']['name']['minLength'] = 3
+sample_dict['properties']['name']['maxLength'] = 5
+sample_dict['additionalProperties'] = False
+st_sample = hypothesis_jsonschema.from_schema(sample_dict)
```

Comment:
I think this should be set in the schema, same as you did for Container.

Reply:
Ok